Content.Server/Body/Systems/BodySystem.cs

```diff
@@ -136,22 +130,15 @@ public override HashSet<EntityUid> GibBody(EntityUid bodyId, bool gibOrgans = fa
 
         _audio.Play(body.GibSound, filter, coordinates, true, audio);
 
-        var containers = GetBodyContainers(bodyId, body: body).ToList();
-
-        foreach (var container in containers)
+        foreach (var entity in gibs)
         {
-            foreach (var entity in container.ContainedEntities)
+            if (deleteItems)
+            {
+                QueueDel(entity);
+            }
+            else
             {
-                if (deleteItems)
-                {
-                    QueueDel(entity);
-                }
-                else
-                {
-                    container.Remove(entity, EntityManager, force: true);
-                    SharedTransform.SetCoordinates(entity,coordinates);
-                    entity.RandomOffset(0.25f);
-                }
+                entity.RandomOffset(0.25f);
```

Comment:
Use SetCoordinates + coordinates.WithOffset for this instead as RandomOffset uses a few deprecated methods please.

Reply:
Like this?
(I couldn't find anything called WithOffset.)

Reply:
I don't think it was the correct one. Did a test with the updates and the parts are still pretty neatly organized 
![image](https://github.com/space-wizards/space-station-14/assets/54602815/3038325e-3fdf-47a1-baf4-429f9c24d205)

Reply:
that looks normal, maybe if you want it messier throw each part a short distance so as its being gibbed it looks better

Reply:
I'm sure gibbing use to at least have the organs & parts rotated and not so neatly displayed.